prime-router/docs/universal-pipeline/route.md

```diff
@@ -1,17 +1,462 @@
-# Route
+# Universal Pipeline Routing
 
-During this step the UP goes through every Full ELR receiver and checks if the message can be routed to them. It goes through the receiver’s quality, jurisdictional, routing and condition filters. If all the filters pass RS will route the message to the receiver and also remove any conditions that don’t match the condition filter.
+# Context
 
-Note: This step occurs after the Convert step and before the Translate step.
+The Route function’s purpose is to match FHIR bundles with receivers. Each receiver connected with ReportStream has unique interests in the data that flows through the pipeline. Routing is designed to find the data that meet those interests.
 
-## Filters
+The Route function follows the convert function. At this point all data will be in FHIR format. These messages are passed to the FHIR Router which first decodes creating a FHIR Bundle. **_FHIRRouter.applyFilters_** does the work to find receivers that accept the bundle. With the list of acceptable receivers, FHIR Endpoints are added to the Provenance resource identifying those receivers.  An endpoint describes the details of a receiver including which test results to include. With that information, the message is passed to the Translate function where receiver specific work is done.
```

Comment:
"decodes creating a FHIR bundle" is confusing. Maybe you just meant to put "decodes a FHIR bundle"?